.github/workflows/pr-tests-stack.yml

```diff
@@ -261,8 +264,9 @@ jobs:
       - name: Get pip cache dir
         if: steps.changes.outputs.stack == 'true'
         id: pip-cache
+        shell: bash
         run: |
-          echo "::set-output name=dir::$(pip cache dir)"
+          echo "dir=$(pip cache dir)" >> %GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "dir=$(pip cache dir)" >> $GITHUB_OUTPUT
```

Reply:
Great work. Thank you! Just 1 small fix.

Reply:
Good catch!